### PR TITLE
Punished sect blessing rework

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request
Instead of punished sect healing people like the normal bibble- you take their burdens on instead!
All bodypart damage and wounds is transferred onto you, as well as blood, suffocation, cloning damage and toxins
The amount of damage you take is dependant on your burden level, decreasing by 7% with every burden level (100% at level 0, 93% at level 1, 37% at level 9 (the point at which you become a psyker))
the punished sect is now an absolute trauma, this makes more sense because aheals shouldnt really clear it more than they clear stuff like paraplegics

## Why It's Good For The Game
I think it's an interesting tweak of letting this chaplain sect do something the others can't in healing, yet at a possibly huge cost.

## Changelog
:cl:
add: Instead of punished sect healing people like the normal bibble- you take their burdens on instead!
/:cl:

### CODE REVIEW

**Comment by GPeckman** [comment]
> cloning damage

I thought that got removed?

File: code/modules/religion/religion_sects.dm [hunk 1]
```diff
@@ -313,6 +316,73 @@
...
+			target.transfer_blood_to(chaplain, target.blood_volume - BLOOD_VOLUME_EXCESS, forced = TRUE)
+	target.update_damage_overlays()
+	chaplain.update_damage_overlays()
+	if(transferred)
+		target.visible_message(span_notice("[chaplain] takes on [target]'s burden!"))
+		to_chat(target, span_boldnotice("May the power of [GLOB.deity] compel you to be healed!"))
+		playsound(chaplain, SFX_PUNCH, 25, vary = TRUE, extrarange = -1)
+		target.add_mood_event("blessing", /datum/mood_event/blessing)
+	else
+		to_chat(chaplain, span_warning("They hold no burden!"))
```
[the remark this answers is not in the document]
Comment (tralezab) [reply code/modules/religion/religion_sects.dm:384]: I agree but would also be fine if it stayed. "no burden!" is the alert version

**Review by tralezab — APPROVED** [review]

File: code/__DEFINES/religion.dm [hunk 2]
```diff
@@ -9,6 +9,8 @@
 #define ALIGNMENT_NEUT "neutral"
 #define ALIGNMENT_EVIL "evil"
 
+#define DEFAULT_SMACK_CHANCE 60
```
Comment (tralezab) [inline code/__DEFINES/religion.dm:12]:
```suggestion

/// the probability, when not overridden by sects, for a bible's bless effect to trigger on a smack
#define DEFAULT_SMACK_CHANCE 60
```

**Comment by tralezab** [comment]
missed that one tidbit but i got it 👍

**Pull request merged**